addons/test_mail/tests/test_mail_message.py

```diff
@@ -45,7 +45,10 @@ def test_empty_message(self):
             'res_id': 0,
             'res_model': 'mail.compose.message',
         })
-        record = self.env['mail.test.track'].create({'name': 'EmptyTesting'})
+        record = self.env['mail.test.track'].create({
+            'name': 'EmptyTesting',
+            'track_hidden_fields': ','.join(self.env['mail.test.track']._fields)
+        })
```

Comment:
Continuing previous comment: would say that if track_hidden_fields is void, behavior is like before so that we don't impact tests.